spec/features/api/v1/places_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+RSpec.configure do |config|
```

Comment:
I'm surprised you need this -- are you sure you do? I would expect it to be in testHelper if so

Reply:
otherwise you'd have to include it at the beginning of every controller test

Reply:
We are unsure we can move this. It has to do with Devise. In our helper we have 

```
Rspec.configure do |config|
```
We do not have (in the helper) the following for Devise
```
Rspec.configure do |config|
            config.include Devise::Test::ControllerHelpers, type: :controller 
end
```